arm/Microsoft.RecoveryServices/vaults/protectionContainers/protectedItems/deploy.bicep

```diff
@@ -1,66 +0,0 @@
-@description('Required. Name of the resource.')
-param name string
-
-@description('Conditional. Name of the Azure Recovery Service Vault Protection Container. Required if the template is used in a standalone deployment.')
-param protectionContainerName string
-
```

Comment:
Also, please attach pipeline runs / badges to show that your changes were tested

Reply:
@prasanjeets you can follow these guidelines to add the badge: [Contribution guide Contribution flow · Azure/ResourceModules Wiki (github.com)](https://github.com/Azure/ResourceModules/wiki/Contribution%20guide%20-%20Contribution%20flow#open-a-pr)

Reply:
Done